### ISSUE TITLE
method "getPrettyXML" of function mxUtils is not a valid method name.

### ISSUE THREAD

**Issue by daiboom** [typed-mxgraph/typed-mxgraph#29]
I tried to write B from A, but it failed. I tried to find out why.

```
// package.json
{
  devDependencies:{
    "@typed-mxgraph/typed-mxgraph": "^1.0.0-2",
    }
  dependencies:{
    mxgraph: "4.2.2"
  }
}
```

![ErrorScreenShot](https://user-images.githubusercontent.com/19921898/108966236-472f9b00-76c1-11eb-88ef-0fdb5c0d20a0.png)

![image](https://user-images.githubusercontent.com/19921898/108966821-04ba8e00-76c2-11eb-872e-98f9a04c7729.png)

![image](https://user-images.githubusercontent.com/19921898/108966904-287dd400-76c2-11eb-9e79-e054564bd258.png)

next screenshot /node_modules/mxgraph/javascript/dist/js/util/mxUtils.js
There is no "getPrettyXML" function in mxUtils. Instead, there is a  "getPrettyXml" function.

![image](https://user-images.githubusercontent.com/19921898/108967325-b8238280-76c2-11eb-9bc7-17b035d786b5.png)

**Comment by hungtcs** [comment]
Hi @Choidaeboem, thanks for your report, this may be a spelling error, I will check and correct as soon as possible.

**Comment by hungtcs** [comment]
Fixed in #30
